modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts

```diff
@@ -100,6 +100,10 @@ export class PageEditorPage {
 	async addFragment(setName: string, name: string, dropTarget?: Locator) {
 		await this.goToSidebarTab('Fragments and Widgets');
 
+		await this.page
+			.getByRole('tab', {exact: true, name: 'Fragments'})
+			.click();
```

Comment:
@pat270 why do we need this?

Reply:
@ethib137 The steps before we add a widget to the page and the widget tab is still active. `addFragment` assumes the Fragment tab will always be active.